boards/msb-430/include/board.h

```diff
@@ -65,6 +68,10 @@ extern "C" {
 #define LED_RED_ON          LEDS_PxOUT &=~LEDS_CONF_RED
 #define LED_RED_OFF         LEDS_PxOUT |= LEDS_CONF_RED
 #define LED_RED_TOGGLE      LEDS_PxOUT ^= LEDS_CONF_RED
+#define LED_GREEN_ON        /* not present */
```

Comment:
You just define the macros so that the preprocessor expands them to a empty string in *main.c* and the compiler can "eat" just a single ';'. Correct?